### ISSUE TITLE
script fails to install in ubuntu 16.04 and 16.10 due to wrong repository/lack of certificate. 

### ISSUE THREAD

**Issue by frustratedpenguin** [jlund/streisand#534]
>TASK [openvpn : Add the official OpenVPN repository] ***************************
task path: /home/ivan/streisand/playbooks/roles/openvpn/tasks/main.yml:8
Using module file /home/ivan/algo-master/env/lib/python3.6/site-packages/ansible/modules/core/packaging/os/apt_repository.py
<45.32.119.52> ESTABLISH SSH CONNECTION FOR USER: root
<45.32.119.52> SSH: EXEC ssh -vvv -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o User=root -o ConnectTimeout=100 -o ControlPath=/home/ivan/.ansible/cp/ansible-ssh-%h-%p-%r 45.32.119.52 '/bin/sh -c '"'"'/usr/bin/python && sleep 0'"'"''                                                                                                                                                                       
>fatal: [45.32.119.52]: FAILED! => {
    "changed": false,                                                                                                                                                                          
    "failed": true,
    "invocation": {
        "module_name": "apt_repository"
    },
    "module_stderr": "OpenSSH_7.4p1, OpenSSL 1.0.2k  26 Jan 2017\r\ndebug1: Reading >configuration data /etc/ssh/ssh_config\r\ndebug1: auto-mux: Trying existing master\r\ndebug2: fd 3 setting O_NONBLOCK\r\ndebug2: mux_client_hello_exchange: master version 4\r\ndebug3: mux_client_forwards: request forwardings: 0 local, 0 remote\r\ndebug3: mux_client_request_session: entering\r\ndebug3: mux_client_request_alive: entering\r\ndebug3: mux_client_request_alive: done pid = 22415\r\ndebug3: mux_client_request_session: session request sent\r\ndebug1: mux_client_request_session: master session id: 2\r\nTraceback (most recent call last):\n  File \"/tmp/ansible_6DCiA5/ansible_module_apt_repository.py\", line 538, in <module>\n    main()\n  File \"/tmp/ansible_6DCiA5/ansible_module_apt_repository.py\", line 527, in main\n    cache.update()\n  File \"/usr/lib/python2.7/dist-packages/apt/cache.py\", line 454, in update\n    raise FetchFailedException(e)\napt.cache.FetchFailedException: W:The repository 'https://build.openvpn.net/debian/openvpn/stable xenial Release' does not have a Release file., W:Data from such a repository can't be authenticated and is therefore potentially dangerous to use., W:See apt-secure(8) manpage for repository creation and user configuration details., E:Failed to fetch https://build.openvpn.net/debian/openvpn/stable/dists/xenial/main/binary-amd64/Packages  server certificate verification failed. CAfile: /etc/ssl/certs/ca-certificates.crt CRLfile: none, E:Some index files failed to download. They have been ignored, or old ones used instead.\ndebug3: mux_client_read_packet: read header failed: Broken pipe\r\ndebug2: Received exit status from master 1\r\n",
    "module_stdout": "",
    "msg": "MODULE FAILURE"
}

I really don't know how to keep going after nearly 12 hours trying to install streisand. 

I tried the workaround below, changing distros and repositories, still same issue. 

>    streisand.yml -> roles: -> delete shadowsocks options
    ufw -> tasks -> main.yml -> delete shadowsocks part
    streisand-mirror -> tasks folder-> delete shadowsocks.yml file && vars folder -> delete shadowsocks.yml files too
    In streisand-mirror -> main.yml (download and verify ) delete the shadowsocks option
    then in current dir/template -> mirror-index.md.j2 file -> delete shadowsocks part
    Done~

**Comment by spythedude** [comment]
Best thing to do to get this working is edit this file

streisand/playbooks/roles/openvpn/tasks/main.yml

look for 

    repo: 'deb https://build.openvpn.net/debian/openvpn/stable {{ ansible_lsb.codename }} main'

Change to 

    repo: 'deb http://build.openvpn.net/debian/openvpn/stable {{ ansible_lsb.codename }} main'

Save, Run streisand again :)

**Comment by hwdsl2** [comment]
The reason for this error is that the SSL certificate for build.openvpn.net has expired a few hours ago (as of 03/05/2017 23:00 UTC).

**Comment by jlund** [comment]
Thanks for the bug report! I just committed a workaround for this.

On a related note, I wrote an open source Slack bot in Ruby that can help keep track of upcoming SSL certificate expirations. It posts daily status reminders in the channel(s) of your choice (and they look pretty fantastic thanks to the work of @SecurityInsanity). It's called [CheckYourSSeLf](https://github.com/instructure/CheckYourSSeLf), and this seems like an appropriate time to toss it a little plug :)
